include/yaclib/coroutine/future_coro_traits.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <yaclib/async/future.hpp>
+#include <yaclib/coroutine/detail/cond_awaiter.hpp>
+#include <yaclib/coroutine/detail/promise_type.hpp>
+#include <yaclib/coroutine/detail/resume_deleter.hpp>
+
+template <typename V, typename E, typename... Args>
+struct yaclib_std::coroutine_traits<yaclib::Future<V, E>, Args...> {
```

Comment:
Fact that `yaclib_std` works is trivial but looks odd to me.

P.S. This is not a call to change something